src/hivpy/hiv_status.py

```diff
@@ -0,0 +1,37 @@
+import operator
+
+import numpy as np
+import pandas as pd
+
+from .sexual_behaviour import selector
+
+
+class HIVStatusModule:
+    hiv_a = -0.00016  # placeholder value for now
+    hiv_b = 0.0128    # placeholder value for now
+    hiv_c = -0.156    # placeholder value for now
+
+    def _prob_HIV_initial(self, age):
+        """Completely arbitrary placeholder function for initial HIV presence in population"""
+        return np.clip(self.hiv_a*age**2 + self.hiv_b*age + self.hiv_c, 0.0, 1.0)
+
+    def initial_HIV_status(self, population: pd.DataFrame):
+        """Initialise HIV status based on age (& sex?)"""
+        """Assume zero prevalence for age < 15"""
```

Comment:
We don't enforce this currently, do we? Or does the placeholder distribution ensure that?